js/search.js

```diff
@@ -155,7 +155,7 @@
 		var queryAlias;
 		if (query in BattleAliases) {
 			queryAlias = toId(BattleAliases[query]);
-			if (query === 'pika' || query === 'bee' || query === 'sab') searchPasses.shift();
+			if (queryAlias.indexOf(query) === 0 && qType === 'pokemon') searchPasses.shift();
```

Comment:
Didn't I say slice and compare would be better than either search or indexOf? :|